schemas/app-sre/tekton-pipeline-template-1.yml

```diff
@@ -0,0 +1,59 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+description: |
+  Properties of tekton pipeline templates deployed in tekton pipelines providers
+  namespaces
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-sre/tekton-pipeline-template-1.yml
+  name:
+    type: string
+  type:
+    type: string
+    enum:
+    - onePerSaasFile
+    - onePerNamespace
+  path:
+    type: string
+  variables:
+    type: object
+oneOf:
+- properties:
+    name:
+      type: string
+      maxLength: 61
```

Comment:
Not sure why the max length is 61, in description we know the hard limit is 63, what are the 2 characters between them?

Reply:
The prefix. Until it's removed, this needs to be 61.